spec/cluster_ht_item_spec.rb

```diff
@@ -64,97 +123,99 @@
       expect(cluster).not_to eq(cluster2)
       expect(Cluster.each.to_a.size).to eq(2)
     end
+
+    context "with HT2 as an update to HT" do
+      let(:update_item) { build(:ht_item, item_id: item.item_id) }
+
+      it "removes the old cluster" do
+        first = described_class.new(item).cluster
+        described_class.new(update_item).cluster
+        expect(Cluster.each.to_a.size).to eq(1)
+        new_cluster = Cluster.each.to_a.first
+        expect(new_cluster).not_to eq(first)
+        expect(new_cluster.ht_items.first).to eq(update_item)
+      end
+    end
```

Comment:
I am not clear how this is testing what it says it is testing. Talk about it monday.

Reply:
I confirmed it does what it says it is doing.

Reply:
FWIW I think this was a test that was there before, but was moved to a different context. One of the changes in behavior in this PR is that "cluster" always does an update if necessary.

Reply:
I didn't understand how update_or_add_htitems could satisfy this test, which had me doubting the meaning of 
`expect(new_cluster).not_to eq(first)`
I will surely forget this again, "default comparison is via the string version of the id," i.e. changes to the ocns are insufficient to pass this test.